index.html

```diff
@@ -0,0 +1,30 @@
+  <!DOCTYPE html>
+  <html>
+    <head>
+      <link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+      <link rel="stylesheet" href="./style.css">
+    </head>
+      <body>
+          <div class="container">
+        <div class="info">X's turn</div>
+        <div class="row">
+          <div class="square" id="1"></div>
+          <div class="square" id="2"></div>
+          <div class="square" id="3"></div>
+        </div>
+        <div class="row">
+          <div class="square" id="4"></div>
+          <div class="square" id="5"></div>
+          <div class="square" id="6"></div>
+        </div>
+        <div class="row">
+          <div class="square" id="7"></div>
+          <div class="square" id="8"></div>
+          <div class="square" id="9"></div>
```

Comment:
Also, IDs should begin with a lowercase letter. Anything else can cause grumpiness in some browsers.